hc/api/tests/test_list_checks.py

```diff
@@ -30,19 +31,23 @@ def setUp(self):
 
     def get(self):
         return self.client.get("/api/v1/checks/", HTTP_X_API_KEY="abc")
-
+    
```

Comment:
Make use of python string comments to actually document what this test is suppose to be testing for